.github/workflows/shfmt.yml

```diff
@@ -0,0 +1,20 @@
+name: Check Shell scripts
+
+on:
+  pull_request:
+    paths:
+      - "**/*.sh"
+
+jobs:
+  shfmt:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - run: docker run -v "$(pwd)":/sh -w /sh peterdavehello/shfmt:3.0.0 shfmt -sr -i 2 -l -w -ci .
```

Comment:
Though it looks pretty safe in this case, I'm not sure the migration should also contains version bump(from v2.6.3 to v3.0.0), thought we can make the changes more clean and clear, separate the migration and the dependency update?